nixos/modules/services/networking/corerad.nix

```diff
@@ -19,7 +13,7 @@ in {
     enable = mkEnableOption "CoreRAD IPv6 NDP RA daemon";
 
     settings = mkOption {
-      type = types.uniq types.attrs;
+      type = types.uniq settingsFormat.type;
```

Comment:
`uniq` no longer needed, since this new type doesn't have weird merging behavior like `types.attrs` :)

Reply:
Fixed.